Sources/StreamChatSwiftUI/ChatChannel/Gallery/ShareButtonView.swift

```diff
@@ -16,8 +17,11 @@ struct ShareButtonView: View {
         Button(action: {
             self.isSharePresented = true
         }, label: {
-            Image(systemName: "square.and.arrow.up")
-                .font(fonts.bodyBold)
+            Image(uiImage: images.share)
```

Comment:
The share icon looks a bit different. It's not bold anymore, and it seems the image ratio is changed.